_includes/desktop-builders-setting.md

```diff
@@ -0,0 +1,61 @@
+If you have turned on the Docker Desktop
+[**Builds** view (beta)](/desktop/use-desktop/builds/), you can use the
+**Builders** tab to inspect and manage builders in the Docker Desktop settings.
+
+### Inspect
+
+To inspect builders, find the builder that you want to inspect and select the
+expand icon. You can only inspect active builders.
+
+The detailed builder view shows:
+
+- Builder status
+- Information about the BuildKit runtime
+- Capabilities and driver type
+- Supported platforms
+- Disk usage
+
+### Select a different builder
+
+The **Selected builder** section displays the selected builder.
+To select a different builder:
+
+1. Find the builder that you want to use under **Available builders**
+2. Open the drop-down menu next to the builder's name.
+3. Select **Use** to switch to this builder.
+
+Your build commands now use the selected builder by default.
+
+### Create a builder
+
+To create a builder, use the Docker CLI. See
+[Create a new builder](/build/builders/manage/#create-a-new-builder)
+
+### Remove a builder
+
+You can remove a builder if:
+
+- The builder isn't your [selected builder](/build/builders/#selected-builder)
+- The builder isn't a [default builder](/build/builders/#default-builder),
+  associated with a Docker context.
+
+To remove a builder:
+
+1. Find the builder that you want to remove under **Available builders**
+2. Open the drop-down menu.
+3. Select **Remove** to switch to this builder.
+
+This removes the builder. If the removed builder used the `docker-container`
+driver, the build cache is also removed.
+
+### Stop and start a builder
+
+Builders that use the
+[`docker-container` driver](/build/drivers/docker-container/)
+run the BuildKit daemon in a container.
+You can start and stop the BuildKit container using the drop-down menu.
+
+Running a build automatically starts the container if it's stopped.
+
+You can't start and stop builders using the `docker`, `kubernetes`, and
+`remote` drivers.
```

Comment:
I don't think you can stop remote builders? 🤔

Reply:
That's what I meant to say with this sentence. Only `docker-container` builders can be stopped.

Reply:
I'll try to clarify